React/Views/RCTView.m

```diff
@@ -184,6 +184,53 @@ - (BOOL)didActivateAccessibilityCustomAction:(UIAccessibilityCustomAction *)acti
   return YES;
 }
 
+- (NSString *)accessibilityValue
+{
+  NSMutableString *value = [NSMutableString stringWithString:@""];
+  NSDictionary *roleDescriptions = @{
```

Comment:
can you mark this and `stateDescriptions` as static so you're not creating these every time `accessibilityValue` is called? Or use a static constant defined outside of this method.